File: make_luaexec.lua
dofile("./make.lua")

if string.find(string.lower(platformInfo()), "unix") then
    file.fileRename("Util", "luaexec")
elseif string.find(string.lower(platformInfo()), "windows") then
    file.fileRename("Util.exe", "luaexec.exe")
end

print("make luaexec ok")

File: make_luaexec.py
import os
import platform
import subprocess

if subprocess.call(['python', 'make.py']) == 0:
    if platform.system().lower().find('windows') >= 0:
        os.rename('Util.exe', 'luaexec.exe')
    else:
        os.rename('Util', 'luaexec')
    print('Make luaexec ok.')
else:
    print('Make luaexec failed.')
